common/src/main/java/com/microsoft/identity/common/internal/providers/oauth2/WebViewAuthorizationFragment.java

```diff
@@ -139,14 +140,11 @@ public void onPageLoaded() {
                         // Inject the javascript string from testing. This should only be evaluated if we haven't sent
                         // an auth result already.
                         if (mAuthResultSent == false && !StringExtensions.isNullOrBlank(mPostPageLoadedJavascript)) {
-                            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.KITKAT) {
-                                mWebView.evaluateJavascript(mPostPageLoadedJavascript, null);
-                            } else {
-                                // On earlier versions of Android, javascript has to be loaded with a custom scheme.
-                                // In these cases, Android will helpfully unescape any octects it finds. Unfortunately,
-                                // our javascript may contain the '%' character, so we escape it again, to undo that.
-                                mWebView.loadUrl("javascript:" + mPostPageLoadedJavascript.replace("%", "%25"));
-                            }
+                            // TODO: #1268511 use EvaluteJavascript after tests are stable.
```

Comment:
Hi @jbzdarkid, could you help review this?

Reply:
Can you explain what part of this wasn't working? I saw no issues with `evaluateJavascript` in the tests that Brian & I ran.

Reply:
In the WIP PR to enable `InteractivePrtApiTests`: https://github.com/AzureAD/microsoft-authentication-library-for-cpp/pull/1836/files#diff-a8ca4d4dabdc4dd14163d2252bf0d0f7db2cdf9bc7627e2e7c4dd61a7a6b09adR269
The webview will stuck at signin page, and after connected devtools, I saw endless requests canceled. Added some captures.
https://identitydivision.visualstudio.com/DevEx/_workitems/edit/1268511/